joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstCreator.scala

```diff
@@ -1101,7 +1102,84 @@ class AstCreator(filename: String, phpAst: PhpFile, fileContent: Option[String],
       case arrayDimFetch: PhpArrayDimFetchExpr if arrayDimFetch.dimension.isEmpty =>
         // Rewrite `$xs[] = <value_expr>` as `array_push($xs, <value_expr>)` to simplify finding dataflows.
         astForEmptyArrayDimAssign(assignment, arrayDimFetch)
-
+      case arrayExpr: (PhpArrayExpr | PhpListExpr) =>
+        val loweredAssignNodes = mutable.ListBuffer.empty[Ast]
```

Comment:
Could you pleasse move all the code for this case into a separate method to keep `astForAssignment` easier to read?